addons/project/models/project_project_stage.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models, _
+from odoo import exceptions, fields, models, _
```

Comment:
```suggestion
from odoo.exceptions import UserError
```
:slightly_smiling_face: